app/controllers/wpcc/your_results_controller.rb

```diff
@@ -1,22 +1,33 @@
 module Wpcc
   class YourResultsController < EngineController
+    attr_accessor :schedule, :salary_frequency
```

Comment:
By the way, is the attribute scheduler not conflict with the scheduler method on the controller?